### PR TITLE
#215

### PR DESCRIPTION
**Pull request by evscott**
A quick and dirty implementation of our priority indicator as a button... Please don't judge me for this. 

When implementing this, I discovered significant issues in how our editable/viewable events forms interact. This is something that could be revisited after this milestone, but this dirty implementation does the trick for now.

### CODE REVIEW

File: src/components/events/event_info/EventInfoForm.js [hunk 1]
```diff
@@ -134,22 +137,37 @@ export class EventInfoForm extends Component {
...
       <Table className="table-hover">
         <tbody>
-          {Object.keys(this.state).map((field) => (
-            <tr key={field}>{this.getField(this.state[field])}</tr>
-          ))}
+          {Object.keys(this.state)
+            .filter((field) => field !== 'priority')
+            .map((field) => (
+              <tr key={field}>{this.getField(this.state[field])}</tr>
+            ))}
```
Comment (gzinck) [inline src/components/events/event_info/EventInfoForm.js]: Ha! Hacky. Kind of gross, but I approve.

File: src/components/events/event_info/fields/EventFieldView.css [hunk 2]
```diff
@@ -28,4 +28,11 @@
...
     opacity: .4;
-}
\ No newline at end of file
+}
+
+.priority-btn {
+    border: none !important;
+    background: none !important;
+    outline:none !important;
+    box-shadow: none !important;
```
Comment (gzinck) [inline src/components/events/event_info/fields/EventFieldView.css]: Yikess!!!! A better approach is to do more specific selectors (say, `div.priority-btn` or `button.priority-btn`) and that sometimes fixes it. Or use an id: `#priority-btn`.

**Review by gzinck — APPROVED** [review]
Looks good! The `!important` is terrible. @morgankbender disapproves. She says she'll be changing it anyways, though. You're good to merge.

File: src/components/events/event_info/EventInfoForm.js (hunk 1 above)
Comment (evscott) [reply src/components/events/event_info/EventInfoForm.js]: Haha yeah... Not thrilled...

File: src/components/events/event_info/fields/EventFieldView.css (hunk 2 above)
Comment (evscott) [reply src/components/events/event_info/fields/EventFieldView.css]: Those are good suggestions :). I suppose I'll leave this to Morgan?

Comment (morgankbender) [reply src/components/events/event_info/fields/EventFieldView.css:37]: Yupp... I will see what I get to anyways lolll

**Pull request merged**